### ISSUE TITLE
Solve Play 2.5 version conflicts

### ISSUE THREAD

**Issue by fugafree** [julienrf/play-jsmessages#47]
Version 2.0.0 supports both Play 2.4 and Play 2.5 but in plugins.sbt Play 2.4 is used.
That couse version conflicts during sbt's dependency resolving:

```
[warn]  * com.typesafe.play:play_2.11:2.5.18 is selected over 2.4.0
[warn]      +- com.typesafe.play:play-cache_2.11:2.5.18           (depends on 2.5.18)
[warn]      +- com.typesafe.play:play-jdbc-api_2.11:2.5.18        (depends on 2.5.18)
[warn]      +- com.typesafe.play:play-logback_2.11:2.5.18         (depends on 2.5.18)
[warn]      +- com.typesafe.play:play-ws_2.11:2.5.18              (depends on 2.5.18)
[warn]      +- com.typesafe.play:play-server_2.11:2.5.18          (depends on 2.5.18)
[warn]      +- org.julienrf:play-jsmessages_2.11:2.0.0            (depends on 2.4.0)
```

**Comment by fugafree** [comment]
And maybe there is an unused ws dependency in 2.0.0? #44
I have an sbt version conflict for that too.

**Comment by julienrf** [comment]
I think the 2.0.0 version is really binary compatible with Play 2.4 and 2.5. But I’m ok to release another version (how would name it, though?) depending on 2.5 to avoid this warning.

**Comment by fugafree** [comment]
Thanks! Better safe than sorry.
Maybe 2.1.x?

**Comment by julienrf** [comment]
would you mind making a PR for that?

**Comment by fugafree** [comment]
Ok, I will do it.
But what about the branching? Will you create a new one for the PR? 2.x.x or 2.1.x?

**Comment by julienrf** [comment]
oh yes, I have to create the target branch now.

**Comment by julienrf** [comment]
You should target the `2.1.x` branch.

**Comment by fugafree** [comment]
I created a PR for this issue but the CI fails for some reason. It want to compile with scala 2.10 which is not supported by play 2.5 anymore.

**Comment by fugafree** [comment]
Fixed by #47.

**Comment by julienrf** [comment]
I published the 2.1.0 version :)

**Comment by fugafree** [comment]
Thanks, and I just added it to my project, everything is okay now, the conflicts are gone.
